great_expectations/checkpoint/actions.py

```diff
@@ -147,6 +137,16 @@ def _get_data_docs_pages_from_prior_action(
         return None
 
 
+def _is_enabled(success: bool, notify_on: Literal["all", "failure", "success"]) -> bool:
```

Comment:
Small but blocking: this looks like it should probably be renamed to something like `_should_notify`.